src/services/cli-http-client.js

```diff
@@ -67,7 +68,8 @@ class CliRequestClient {
     const componentInfo = (headers['User-Agent'] || '').replace(' (', '|').replace(')', '').split('|');
     componentInfo.push(`${os.platform()} ${os.release()} ${os.arch()}`);
     componentInfo.push(this.commandName);
-    headers['User-Agent'] = `${pkg.name}/${pkg.version} (${componentInfo.join(', ')})`;
+    componentInfo.push(this.keytarWord);
+    headers['User-Agent'] = `${pkg.name}/${pkg.version} (${componentInfo.filter(Boolean).join(', ')})`;
```

Comment:
Is there a test which verifies that empty pieces don't get added?

Reply:
there is currently no test for the user-agent header itself in unit tests. I've manually tested in the release feature branch with API keys in the file that the empty piece doesn't get added. [DEBUG] User-Agent: @twilio/cli-core/5.24.0 (twilio-node/3.64.0, node.js v16.2.0, darwin 20.5.0 x64, debugger:logs:list)